src/WorkItems/WorkItems.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0</TargetFrameworks>
```

Comment:
>TargetFrameworks [](start = 5, length = 16)

Here is a scenario that I'm able to repro multiple times:

If my extension targets .NET Core 2.1, creating work items succeeds.

If my extension targets .NET Core 3.1, I'm getting the below exception, which I'm currently investigating. 
ystem.Reflection.TargetInvocationException
  HResult=0x80131604
  Message=Exception has been thrown by the target of an invocation.
  Source=System.Private.CoreLib
  StackTrace:
   at System.RuntimeTypeHandle.CreateInstance(RuntimeType type, Boolean publicOnly, Boolean wrapExceptions, Boolean& canBeCached, RuntimeMethodHandleInternal& ctor)
   at System.RuntimeType.CreateInstanceSlow(Boolean publicOnly, Boolean wrapExceptions, Boolean skipCheckThis, Boolean fillCache)
   at System.Activator.CreateInstance(Type type, Boolean nonPublic, Boolean wrapExceptions)
   at Microsoft.CodeAnalysis.Sarif.WorkItems.SarifWorkItemContext.PopulateWorkItemModelTransformers() in D:\dev\sarif-sdk\src\Sarif.WorkItems\SarifWorkItemContext.cs:line 89

Inner Exception 1:
FileNotFoundException: Could not load file or assembly 'System.Net.Http, Version=4.2.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified.

Reply:
That's COR_E_TARGETINVOCATION. It's not surprising that we don't have forward compatibility, I think. You should target 2.1 for now. After we get this in, we can update to 3.1 and try again.

---
In reply to: [382165181](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r382165181) [](ancestors = 382165181)

Reply:
Got it. Going back to 2.1 and it works.

---
In reply to: [383490884](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r383490884) [](ancestors = 383490884,382165181)